README.md

```diff
@@ -142,7 +148,7 @@ This is a type for representing string values.
 The `get` method returns the value of the `StringType` instance.
 
 ```javascript
-var stringType = new minim.StringType('foobar');
+var stringType = new minim.StringType({}, {}, 'foobar');
```

Comment:
The need to include meta/attributes in the constructor was mainly for consistency sake, but it makes for a worse experience in my opinion. Ideas?